templates/default/49-remote.conf.erb

```diff
@@ -1,8 +1,15 @@
 # Generated by Chef
-$ActionQueueType LinkedList # use asynchronous processing
-$ActionQueueFileName srvrfwd # set file name, also enables disk mode
-$ActionResumeRetryCount -1 # infinite retries on insert failure
-$ActionQueueSaveOnShutdown on # save in-memory data if rsyslog shuts down
+$ActionResumeInterval 10
+$ActionQueueSize 100000
+$ActionQueueDiscardMark 97500
+$ActionQueueHighWaterMark 80000
+$ActionQueueType LinkedList
+$ActionQueueFileName papertrailqueue
```

Comment:
I /think* this needs to be authorized in the apparmor profile or it fails?

```
rsyslogd-2027: error starting up disk queue, using pure in-memory mode [try http://www.rsyslog.com/e/2027 ]
```

Reply:
(or if there is no apparmor profile, maybe the file needs to be writable by whatever user rsyslogd runs as)

Reply:
Ah ok, I'll just revert it to what it was, then.